src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -82,18 +107,26 @@ public PersonCard(Person person, int displayedIndex) {
         person.getNonEmergencyTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
-        StringBinding notesButtonText = Bindings
```

Comment:
This wasn't detected as a merge conflict by GitHub, but it definitely is one. This modification would roll back the changes in https://github.com/AY2324S1-CS2103T-T13-2/tp/pull/110, which we don't want. Maybe try merging master to this branch!

Reply:
thanks for pointing this out! apparently when I merged into the master branch again, seems like it doesn't detect merge conflict so I changed it manually! Fixed in d6c95a1